src/errors.rs

```diff
@@ -297,7 +297,7 @@ error_chain! {
         }
         ToolchainNotSelected {
             description("toolchain is not selected")
-            display("no override and no default toolchain set")
+            display("no override and no default toolchain set; run 'rustup default [toolchain]' to select one")
```

Comment:
I don't like the word 'select' here, I think it implies showing a menu - which it won't (but we do on windows during first install kindof).

I can suggest two alternatives to move this forward.

1. s/select/set/
2. extend the places this is used to determine the default toolchain for the platform, and then pass that into the error, so that it can become a copy-and-pastable valid command.

Reply:
1. Sure
2. If there is no default toolchain set, can we really get a default toolchain ? Maybe we should just suggest `rustup default stable`.

Reply:
Sure, though stable is just shorthand for stable-{host-triple}

Reply:
I changed it. I don't think it is worth to suggest typing the whole host triple.